CHANGELOG/v1.4.md

```diff
@@ -154,3 +166,8 @@ Other minor additions
   ```
 
 * Added module `≤-Reasoning` to `Data.Rational.Unnormalised.Properties`
+
+* Added the function `lift₀` to `Data.Fin.Permutation` which takes a
```

Comment:
I would just add the type here rather than a description, and you also need to add the types of the other proofs you've added.